airbyte-integrations/connectors/source-surveycto/metadata.yaml

```diff
@@ -2,7 +2,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: dd4632f4-15e0-4649-9b71-41719fb1fdee
-  dockerImageTag: 0.1.1
+  dockerImageTag: 0.1.2
```

Comment:
missing the Dockerfile inside the connector folder.

Reply:
@marcosmarxm done now..